src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.product-card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+  transition: $transition-duration;
+
+  &__image {
+    display: block;
+    margin: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-top: 16px;
```

Comment:
According to the code style rules, 'align-items' property should be written before 'justify-content'. Please reorder the properties.